frontend/src/modules/incident-filing/components/IncidentFormInternal.js

```diff
@@ -528,6 +528,7 @@ class IncidentFormInternal extends Component {
                                                     type="datetime-local"
                                                     value={values.occured_date}
                                                     InputLabelProps={{ shrink: true }}
+                                                    defaultValue={moment().format("YYYY-MM-DDTHH:mm")}
```

Comment:
values not set on formik props, which then validates as datetime is not selected.

Reply:
Fixed by using setting current date at Formik initialization when there is no previous record.